builtin/logical/aws/path_user.go

```diff
@@ -130,6 +131,11 @@ func pathUserRollback(ctx context.Context, req *logical.Request, _kind string, d
 		MaxItems: aws.Int64(1000),
 	})
 	if err != nil {
+		// If the user isn't found, let's just move on. This could happen if we fail
+		// to create the new user in the first place
```

Comment:
Ah, thank you, that's very true. I added this as an aside while I was developing #5216 , I had IAM Users that legitimately failed to create and were causing lots of noisy error logs here because the user never did exist, so I added this to clean up and move on. 

I'll remove this next chance I get (tomorrow likely)

Reply:
Removed in 9c39bc0420c0357483d0f108fdf9292caf0f7bd7